packages/fxa-payments-server/src/routes/Product/PlanDetails/index.tsx

```diff
@@ -15,7 +15,9 @@ const availableDetails: availableDetailsType = {
   // Examples:
   // '123doneProProduct': React.lazy(() => import('./Details123donePro')),
   // '321doneProProduct': React.lazy(() => import('./Details321donePro')),
-  plan_F4bof27uz71Vk7: React.lazy(() => import('./Details123DonePro')),
+  'fortressProProduct': React.lazy(() => import('./Details123DonePro')), // same template
```

Comment:
Looks like`fortressProProduct` (a plan ID) here should be `fortressProMonthly` (a plan ID)